samples/excel/89-preview-apis/events-worksheet-changed.yaml

```diff
@@ -0,0 +1,177 @@
+order: 9
+id: excel-events-worksheet-changed
+name: Events - Worksheet changed
+description: Add event handlers on the data changed and worksheet added events
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#egister-on-data-changed-handler").click(() => tryCatch(registerOnDataChangedHandler));
+        $("#data-changed").click(() => tryCatch(changeData));
+        $("#register-on-add-handler").click(() => tryCatch(registerOnAddHandler));
+        $("#add-worksheet").click(() => tryCatch(addWorksheet));
+
+        async function registerOnDataChangedHandler() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getItem("Sample");
+                sheet.onDataChanged.add(onDataChange);
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("A handler has been registered for the onDataChanged event.");
+            });
+        }
+
+        async function changeData() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getItem("Sample");
+                let range = sheet.getRange("B5");
+                range.values = [[800]];
+                range.format.autofitColumns();
+
+                await context.sync();
+
+                console.log("B5 value has been changed.");
+            });
+        }
+
+        async function onDataChange(event) {
+            await Excel.run(async (context) => {
+                console.log("Handler for data changed event has been triggered.  Data changed address : " + event.address);
```

Comment:
Let's consistently use the official name everywhere including page text, code comments, and button labels: OnDataChanged event.